services/claimsrv/service.go

```diff
@@ -29,6 +29,7 @@ type Service interface {
 		claimAuthorizeKSignSecp256k1 core.ClaimAuthorizeKSignSecp256k1) error
 	AddUserIdClaim(idAddr common.Address, claimValueMsg ClaimValueMsg) error
 	AddDirectClaim(claim core.ClaimBasic) error
+	AddLinkObjectClaim(claim core.ClaimLinkObjectIdentity) error
```

Comment:
This is a bit strange.  The rest of the AddClaim functions take the parameters of the claim as arguments whereas AddLinkObjectClaim and AddDirectClaim take the claim itself.